profiles/student/nlMeminger.yaml

```diff
@@ -0,0 +1,10 @@
+---
+name: Nathaniel E. Larrimore
+site: nathaniellarrimore.com
```

Comment:
This should be `website` instead of `site`. See the [example profile](https://github.com/FOSSRIT/FOSSProfiles/blob/master/example_profile.yaml) as an example.

Also, please add either `http://` or `https://` since this goes directly into a `<a href="">` tag.